views/form.blade.php

```diff
@@ -14,9 +14,12 @@
             @if(!$tabObj->isEmpty())
                 @include('admin::form.tab', compact('tabObj'))
```

Comment:
http://prntscr.com/euj9gy 
It's work, simple.
[Demo horizontal fields](https://github.com/SleepingOwlAdmin/demo/blob/master/admin/Http/Sections/Contacts5.php#L116)